docs/skopeo-delete.1.md

```diff
@@ -15,6 +15,7 @@ The effect of this is registry-specific; many registries don’t support this op
 
 **WARNING**: If _image-name_ contains a tag (but not a digest), in the current version of Skopeo this resolves the tag into a digest, and then deletes the manifest by digest, as described above (possibly deleting all tags pointing to that manifest, not just the provided tag). This behavior may change in the future.
 
+**WARNING**: If _image-name_ is a list manifest (ex: multi-platform images), only the `list.manifest.json` is deleted which may lead to storage issue.  Only the registry server is able to find dandling manifests as one may be referenced by many `list.manifest.json`. Some registries like artifactory don't have any garbage collector built in.
```

Comment:
`list.manifest.json` is not a widely-used terminology, to my knowledge. “A multi-platform manifest list” maybe.